package.json

```
@@ -69,6 +69,7 @@
     "@kodadot1/sub-api": "0.2.0-rc.0",
     "@nuxtjs/apollo": "5.0.0-alpha.6",
     "@nuxtjs/i18n": "8.0.0-rc.3",
+    "@nuxtjs/proxy": "^2.1.0",
```

we doesn't need this one, use `routeRules` or `nitro` config

```suggestion
    "@nuxtjs/proxy": "^2.1.0",
```